### PR TITLE
Added `IncludeOpenAPIAnalyzers` flag to the produced NuGet package

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
Hi Rico.

In ASP.NET Core we've added a feature to the WebSDK to enable OpenAPI analyzers when the `IncludeOpenAPIAnalyzers` flag is present. This change adds that flag to the NSwag.AspNetCore project, so that as soon as that NuGet package is referenced from an ASP.NET Core project, the API Analyzer is enabled.

/cc @pranavkm 

Addresses https://github.com/aspnet/AspNetCore/issues/8418

### CODE REVIEW

File: src/NSwag.AspNetCore/NSwag.AspNetCore.csproj [hunk 1]
```diff
@@ -22,6 +22,9 @@
     <EmbeddedResource Include="SwaggerUi\**\*" Exclude="bin\**;obj\**;**\*.xproj;packages\**;@(EmbeddedResource)" />
     <EmbeddedResource Include="SwaggerUi3\**\*" Exclude="bin\**;obj\**;**\*.xproj;packages\**;@(EmbeddedResource)" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="build\NSwag.AspNetCore.props" />
```
Comment (pranavkm) [inline src/NSwag.AspNetCore/NSwag.AspNetCore.csproj]: This should be in `build\netcoreapp3.0`

File: src/NSwag.AspNetCore/build/NSwag.AspNetCore.props [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project>
+  <ItemGroup>
```
Comment (pranavkm) [inline src/NSwag.AspNetCore/build/NSwag.AspNetCore.props]:
```xml
<Project>
  <PropertyGroup>
    <IncludeOpenAPIAnalyzers>true</IncludeOpenAPIAnalyzers>
  </PropertyGroup>
</Project>
```

I'd remove the xml header. Also needs to be a property

**Comment by mkArtakMSFT** [comment]
Thanks @dougbu. I'm closing this PR, as yours now includes the flag.

**Pull request closed without merging**